homeassistant/components/intellifire/__init__.py

```diff
@@ -0,0 +1,41 @@
+"""The IntelliFire integration."""
+from __future__ import annotations
+
+from intellifire4py import IntellifireAsync
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN, LOGGER
+from .coordinator import IntellifireDataUpdateCoordinator
+
+PLATFORMS: list[str] = [Platform.BINARY_SENSOR]
```

Comment:
The type is inferred here, so no need to type it.

```suggestion
PLATFORMS = [Platform.BINARY_SENSOR]
```